bower.json

```diff
@@ -23,6 +23,7 @@
   },
   "devDependencies": {
     "purescript-assert": "^4.1.0",
-    "purescript-console": "^4.2.0"
+    "purescript-console": "^4.2.0",
+    "purescript-exceptions": "^4.0.0"
```

Comment:
Since `exceptions` is used in the library code this would need to be under the `dependencies` key, not `devDependencies`.

Reply:
Whoops, I misread this — it looks like this is only used in the test code, so you were right putting that in dev dependencies in the first place.